iOS_SDK/OneSignalSDK/UnitTests/UnitTests.m

```diff
@@ -1193,17 +1193,17 @@ - (void)testOpeningWithAdditionalData {
  */
 - (void)receivedCallbackWithButtonsWithUserInfo:(NSDictionary *)userInfo {
     __block BOOL receivedWasFire = false;
-    [UnitTestCommonMethods initOneSignalWithHanders_andThreadWait:^(OSNotification *notification) {
+    [UnitTestCommonMethods initOneSignalWithHanders_andThreadWait:^(OSNotificationGenerationJob *notifJob) {
         receivedWasFire = true;
-        let actionButons = @[ @{@"id": @"id1", @"text": @"text1"} ];
-        XCTAssertEqualObjects(notification.payload.actionButtons, actionButons);
+        //let actionButons = @[ @{@"id": @"id1", @"text": @"text1"} ];
```

Comment:
Looks like you commented out do to `OSNotificationGenerationJob` not having `actionButtons` on it. Let's leave a TODO to fix test before the 3.0.0 release. We may want to revisit the `OSNotificationGenerationJob` class structure and make changes before the release.

Reply:
added the todo